src/tests/ReactA11yAnchorsRuleTests.ts

```diff
@@ -259,6 +261,30 @@ describe('reactA11yAnchorsRule', () : void => {
         TestHelper.assertViolations(ruleName, script, [ ]);
     });
 
+    it('should pass when identical hrefs have texts with different cases on ignore-case enabled', () : void => {
```

Comment:
Good unit tests for the case of the rules enabled! But: could you please also add equivalent tests for the same cases but with your new options not enabled? That way we still capture the case of the rule accidentally always enabling those options.

Reply:
Good idea! I've added the new unit tests on the latest commit.